packages/desktop/components/popups/AddNodePopup.svelte

```diff
@@ -5,12 +5,14 @@
     import { closePopup } from '@auxiliary/popup'
     import { activeProfile } from '@core/profile'
     import { showAppNotification } from '@auxiliary/notification'
+    import { Platform } from 'shared/lib/core/app'
+    import { registerProposalsFromNode } from 'shared/lib/contexts/governance'
 
     export let node: INode = EMPTY_NODE
     export let isEditingNode: boolean = false
     export let onSuccess: (..._: any[]) => void
 
-    const currentNode = node
+    const currentNode = { ...node }
```

Comment:
do you do this to copy the object without reference? 
if yes, you would need a deep copy, eg. `JSON.parse(JSON.stringify())`

Reply:
Forgot that we have a `deepCopy` util function that does this in `core/utils/object.ts`